installations/InstallationsSnippets/ObjCSnippets.m

```diff
@@ -20,6 +20,18 @@
 
 @implementation ObjCSnippets
 
+- (void)handleInstallationIDChange {
+  // [START handle_installation_id_change]
+  [[NSNotificationCenter defaultCenter] addObserverForName: FIRInstallationIDDidChangeNotification
+                                                    object:nil
+                                                     queue:nil
+                                                usingBlock:^(NSNotification * _Nonnull notification) {
+    // Fetch new Installation ID
+    [self fetchInstallationsID];
+  }];
+  // [END handle_installation_id_change]
+}
```

Comment:
Basically the same comments as for Swift snippet.

Reply:
done